coreos-installer

```diff
@@ -633,6 +633,18 @@ write_image_to_disk() {
     udevadm settle
 }
 
+############################################################
+#Perform a HTTP POST to URL to signal that the install is complete
+############################################################
+call_home() {
+    CALLHOME_URL=$(cat /tmp/callhome_url)
+    if [ ! -z "$CALLHOME_URL" ]
+      echo "Calling home: $CALLHOME_URL" >> /tmp/debug
+      curl -k -o /dev/null --data-binary @/tmp/debug "$CALLHOME_URL" >>/tmp/debug 2>&1
```

Comment:
I always like using long versions in scripts to see what's going on. So `--insecure` instead of `-k`